### ISSUE TITLE
Shifted in Meat

### ISSUE THREAD

**Issue by Andom325** [dalaranwow/dalaran-wow#5199]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Well tanking in scholomance as a druid in bear form.

I was shape shifted into a pile of meat and can not get out of it.
I have tried to shape shift with food (delights) and i become a pirate for one hour but when the food timer expires i turn back into a pile of meat.
When i use the druid ability to shift into bear or cat form i get my ability's but still look like meat.

I have tried:
1. Using druid shape shifting, i can even Fly as meat.
2. i have gotten killed and i comeback with this issue persisting.
3. I can not ride a mount or take flights because of being shifted.
4. logging out and back in did not help.

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. i can not undo this issue so i have no idea how to duplicate it
<img width="960" alt="001" src="https://user-images.githubusercontent.com/34797154/34321840-94865a7e-e7cc-11e7-82b2-965174eaf627.png">
<img width="960" alt="002" src="https://user-images.githubusercontent.com/34797154/34321841-94b0971c-e7cc-11e7-997d-0fda962bf87c.png">
<img width="960" alt="003" src="https://user-images.githubusercontent.com/34797154/34321842-94c7f4a2-e7cc-11e7-9188-c47bf46eff0c.png">
<img width="960" alt="004" src="https://user-images.githubusercontent.com/34797154/34321843-94de897e-e7cc-11e7-8cb1-c31089360ace.png">

2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
Attached screen shots, but if you want to see it, find me in game.

**Comment by Andom325** [comment]
Still shifted after 4 days, hard to play this way.

**Comment by Lothloryen** [comment]
Hello there,

You should be fine already (write a ticket in order to have such things resolved though, please). I thought this issue was already fixed, I will take a look to see why it is happening again.

Thank you very much.

**Comment by Andom325** [comment]
thank you for the quick fix,
i thought this was the ticketing system,
i did open an in game ticket for this, but it looked more like a Dev issue.
Logged in and i can see my Cow like body again,

Thank you!
